app/components/questions/TaxiPaymentMethod.js

```diff
@@ -0,0 +1,115 @@
+import { useState, useEffect } from 'react'
+import useFetch from 'use-http'
+import { Table } from '../../components/Table'
+import { Notification } from '../../components/Notification'
+import {
+  Formatter
+} from '../../site/metrics'
+
+const COMMON_PAYMENT_ENDPOINT = `${process.env.NEXT_PUBLIC_API}/question/taxitrips`;
+
+const COMMON_PPAYMENT_COLS = [
```

Comment:
Small comment on typo: "ppayment" It's not a problem because you have it in both places.

Reply:
Nevermind, I see this stands for pickup_payment, not a typo. Can you write out pickup and dropoff for these?